modules/tracker/controllers/components/ApisubmissionComponent.php

```diff
@@ -0,0 +1,266 @@
+        $apihelperComponent->requirePolicyScopes(array(MIDAS_API_PERMISSION_SCOPE_ADMIN_DATA));
+
+        $this->_checkUser($args,
+                          'Only authenticated users can delete submissions.');
+
+        /** @var int $submissionId */
+        $submissionId = $args['id'];
+
+        /** @var Tracker_SubmissionModel $submissionModel */
+        $submissionModel = MidasLoader::loadModel('Submission',
+                                                  $this->moduleName);
+
+        /** @var Tracker_SubmissionDao $submissionDao */
+        $submissionDao = $submissionModel->load($submissionId);
+
+        if (is_null($submissionDao) || $submissionDao === false) {
+            throw new Exception('A submission with id '. $submissionId.
+                                ' does not exist.', MIDAS_NOT_FOUND);
+        }
+
+        $submissionModel->delete($submissionDao);
+    }
+
+    /**
+     * Retrieve the given submission.
+     *
+     * @path /tracker/submission/{id}
+     * @http GET
+     * @param id
+     * @return array
+     *
+     * @param array $args parameters
+     * @throws Exception
+     */
+    public function get($args)
+    {
+        /** @var ApihelperComponent $apihelperComponent */
+        $apihelperComponent = MidasLoader::loadComponent('Apihelper');
+        $apihelperComponent->validateParams($args, array('id'));
+        $apihelperComponent->requirePolicyScopes(
+            array(MIDAS_API_PERMISSION_SCOPE_READ_DATA));
+
+        /** @var int $submissionId */
+        $submissionId = $args['id'];
+
+        /** @var Tracker_SubmissionModel $submissionModel */
+        $submissionModel = MidasLoader::loadModel('Submission',
+                                                  $this->moduleName);
+
+        /** @var Tracker_SubmissionDao $submissionDao */
+        $submissionDao = $submissionModel->load($submissionId);
+
+        if (is_null($submissionDao) || $submissionDao === false) {
+            throw new Exception('A submission with id '. $submissionId.
+                                ' does not exist.', MIDAS_NOT_FOUND);
+        }
+
+        return $this->_toArray($submissionDao);
+    }
+
+    /**
+     * TODO.
+     *
+     * @path /tracker/submission
+     * @http GET
+     * @return array
+     *
+     * @param array $args parameters
+     * @throws Exception
+     */
+    public function index($args)
+    {
+        /** @var ApihelperComponent $apihelperComponent */
+        $apihelperComponent = MidasLoader::loadComponent('Apihelper');
+        $apihelperComponent->validateParams($args, array());
+        $apihelperComponent->requirePolicyScopes(
+            array(MIDAS_API_PERMISSION_SCOPE_READ_DATA));
+
+        // TODO: Implement index().
+
+        return array();
+    }
+
+    /**
+     * Create a new submission.
+     *
+     * @path /tracker/submission
+     * @http POST
+     * @param producer_id
+     * @param uuid (Optional)
```

Comment:
Do we want to validate the format of the uuid (should match the output of `UuidComponent`). Also document the format.